Add --skip_sex_check parameter (#957)
genomic-medicine-sweden/nallo: docs/usage.md

#### Alignment

The majority of subworkflows depend on the alignment subworkflow which requires `--fasta`.

| Parameter | Description                                                                                                                                                                        |
| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `fasta`   | Reference genome, either gzipped or uncompressed (e.g. [GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz](https://lh3.github.io/2017/11/13/which-human-reference-genome-to-use)) |

Turned off with `--skip_alignment`.

#### Sex check

> Some subworkflows are sex-dependent and need a sex-inference step if the metadata is 'unknown'. This step requires `--somalier_sites`.

Reviewer: ```suggestion
Some subworkflows are sex-dependent and need a sex-inference step if samples are input with unknown sex. This step requires `--somalier_sites`.
```
